package.json

```
@@ -6,6 +6,7 @@
   "scripts": {
     "format": "prettier --write '{pages,pageTests,src}/**/*.{js,css,md,yml}'",
     "format:check": "prettier --check '{pages,pageTests,src}/**/*.{js,css,md,yml}'",
+    "eslint": "eslint . --ext .js",
```

Wondering if this command should be a generic name instead of the name of the linter? Thinking if we wanted to swap out ESLint for another we'd just change the command instead of both the command and script name.

```suggestion
    "lint": "eslint . --ext .js",
```

---

YES :raised_hands: thank you